sympy/core/relational.py

```diff
@@ -1075,9 +1087,72 @@ def _n2(a, b):
             return dif
 
 
+@dispatch(Rational, Pow)
+def _eval_is_ge(rat, rhs): # noqa:F811
+    if rat.is_zero:
+        return
+    base, exp = rhs.as_base_exp()
+    if base.is_nonnegative and (exp + 1).is_nonpositive:
+        return is_ge(base, rat)
```

Comment:
Here `exp + 1` creates a new Basic instance. It would be better to use `is_ge(exp, 1)`.